projects/js-packages/charts/src/index.ts

```diff
@@ -36,3 +37,4 @@ export type { LineStyles, GridStyles, EventHandlerParams } from '@visx/xychart';
 
 export type { RenderLineStartGlyphProps } from './components/line-chart/line-chart';
 export type { LegendProps, BaseLegendProps, ChartLegendOptions } from './components/legend';
+export type { ConversionFunnelChartProps, FunnelStep } from './components/conversion-funnel-chart';
```

Comment:
Just a note:

if these types are exported from the component itself then they should be accessible by,

```
import type { ConversionFunnelChartProps } from `@automattic/charts/conversion-funnel-chart`
```

It's meant for easier discovery of types and chart related helpers 😄 

cc/ @Automattic/red